packages/auth/src/index.ts

```diff
@@ -302,9 +307,9 @@ export function createAuth<ListTypeInfo extends BaseListTypeInfo>({
         ...keystoneConfig.lists,
         [listKey]: { ...listConfig, fields: { ...listConfig.fields, ...fields } },
       },
-      extendGraphqlSchema: existingExtendGraphQLSchema
-        ? schema => existingExtendGraphQLSchema(extendGraphqlSchema(schema))
-        : extendGraphqlSchema,
+      extendGraphqlSchema: schema => {
+        return extendGraphqlSchema(authExtendGraphqlSchema(schema));
+      },
```

Comment:
Not part of this pull request, but is equivalent to the same pattern we used elsewhere